### ISSUE TITLE
add UI clarity for relationship of projects and phase assets on the project detail page

### ISSUE THREAD

**Issue by smcalilly** [fpdcc/ccfp-asset-dashboard#277]
the project detail page shows a map with phase assets. this is kind of confusing because a project has different phases, and the assets can be different per page. does the current project detail show all of those assets, or only the ones for a different phases? ideally the map would show the assets from all phases with some sort of visual indicator/way to distinguish the different assets per page.

this came from a comment by @xmedr in the documentation:

> how do we feel about the edit assets link being in the project detail page? I feel like it might be confusing since it's not clear which phase's assets you're about to edit, and it just takes you to the assets for the first listed phase in the table above it. maybe we remove that link from the project page and/or have some kind of indication on this page of which phases have assets, or which assets are for which phases. just spitballing

**Comment by jim-z** [comment]
@xmedr I agree, this is still confusing and is tied in with issue #289.  Edit asset ability should probably be removed from main description page to force asset editing through phase pages only.  Would be nice if the phase could be added to table here and possibly an asset export button that kicks out geojson or something?

![Image](https://github.com/user-attachments/assets/42f71238-8bf7-4f81-9651-54d1672f425c)
